stable/kubewatch/values.yaml

```diff
@@ -45,6 +45,10 @@ resourcesToWatch:
   pod: true
   job: false
   persistentvolume: false
+  ingress: false
+  namespace: false
+  secret: false
+  configmap: false
```

Comment:
Please update the `values-production.yaml` too

Reply:
done

Reply:
done